src/amo/components/Categories/index.js

```diff
@@ -106,16 +106,6 @@ export class CategoriesBase extends React.Component<InternalProps> {
     }
     const classNameProp = classnames('Categories', className);
 
-    if (!errorHandler.hasError() && !loading && !categories.length) {
```

Comment:
I don't think we should remove this `if` block because it seems unrelated (it handles the case of an empty API response, which might happen).

Reply:
Ah ok - I thought this was the block to validate the addon type but I wasn't sure.